data/avro-schema/src/main/resources/avro/net/corda/data/interop/InteropState.avsc

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "record",
+  "name": "InteropState",
+  "namespace": "net.corda.data.interop",
+  "fields": [
+    {
+      "name": "stateId",
+      "type": "string",
+      "doc": "The key on which inbound events should be forwarded to on the interop event topic."
+    },
+    {
+      "name": "expiryTime",
+      "type": [
+        "null",
+        "long"
+      ],
+      "doc": "The time at which this state will be set to null. This is only set for closed sessions."
+    },
+    {
+      "name": "status",
+      "doc": "The status of a session. VALID for ongoing sessions. INVALID for sessions which are finished and the expiryTime has been set.",
+      "type": {
+        "type": "enum",
+        "name": "InteropStateType",
+        "symbols": [
+          "VALID",
+          "INVALID"
+        ]
+      }
+    },
+    {
+      "name": "aliasHoldingIdentity",
```

Comment:
I think the best would be to store groupId as well, in a as separate field. StateProcessor would be completely independent from a message coming back from `FlowProcessor` in terms of deriving alias groupId (as it would store it previously) and no need to get groupId from the destination field.

Reply:
I would not expect that the destination alias will have a correct alias group name, we haven't really finished implementation inside FlowProcessor. Also saving alias group ID here, allows us to store the full context , not a half of the context. It's better for Session management to be fully independent, and we can also detect any incompatibilities (e.g. bug in FlowProcessor) when a message comes back (crucial in early development stage). Also from admin point of view, imagine at later stage investigating a queue to troubleshooting etc - we can se any Identity names but we have no clue what groups they belong for.